### DISCUSSION TITLE
Can I get the Timestamp (@t) as a unix timestamp?

### DISCUSSION THREAD

**Discussion by kberridge** [datalust/seq-tickets#2088] [Help] [answered]
I used [Seq.App.HttpRequest](https://github.com/datalust/seq-app-httprequest) to send a write to influxdb, it works great!!  The only thing I'm missing is including the timestamp.  I can't simply do {@t} because influx wants a unix timestamp.  Can I get @t formatted as a unix timestamp from Seq somehow?

**Comment by nblumhardt** [comment, marked as answer]
That's an awesome use case for the HTTP request integration :-)

The answer _should_ be:

```sql
Round(TotalMilliseconds(now() - DateTime('1970-01-01T00:00:00Z')), 0)
```

but unfortunately `Seq.Syntax`, which drives `Seq.App.HttpRequest`, [currently lacks several time functions](https://github.com/datalust/seq-app-mail/blob/dev/src/Seq.Syntax/Expressions/Runtime/RuntimeOperators.cs) including `TotalMilliseconds()` and `DateTime()`.

The code required is (I think) fairly trivial, so I'll raise an issue from this discussion and aim to PR a solution early next week :-)

**Reply by nblumhardt** [reply]
https://github.com/datalust/seq-tickets/issues/2090

**Reply by kberridge** [reply]
Thanks!

I think it's worth mentioning, after I posted this, I hit another snag with using Seq.App.HttpRequest to send data to InfluxDb.  The Influx Line protocol is picky about spaces.  It requires you to escape spaces in Tags with `\`!  I would need something like `Replace(MyProperty, " ", "\ ")` which I don't think exists?

So, I think I'll try to write a custom App which looks easy enough.

Thanks again for your help, and for Seq, which I love!

**Reply by nblumhardt** [reply]
Sounds good - thanks @kberridge!
